packages/adapter/src/lib/adapter/adapter.ts

```diff
@@ -7589,17 +7589,26 @@ export class AdapterClass extends EventEmitter {
     registerNotification<Scope extends keyof ioBroker.NotificationScopes>(
         scope: Scope,
         category: ioBroker.NotificationScopes[Scope] | null,
-        message: string
+        /** Message for messangers */
+        message: string,
+        /** Message to be shown in admin */
+        actionData?: ioBroker.NotificationAction
```

Comment:
I think this is general additional data could also contain data later on used by any other adapter

Reply:
Do not understand, what to do here

Reply:
What I mean is it should likely not be something like message shown in admin more like general additional notification data, we could also store stuff here for e.g. messaging adapters later on. Maybe we could also think about making it then like `actionData.ui` where the admin (ui) information is stored.

Reply:
ui... or even admin like `actionData.admin`. No one else can show this info.

Reply:
admin is also fine for me but in theory other UIs could in the future, also if it likely won't ever happen json config is reusable^^

Reply:
I would also propose to name it more general like additionalData or just data or contextData because we could also see that it provides more context for processing ... or messageData because in fact it is sent as a message ... ?!